legal-api/migrations/alembic.ini

```diff
@@ -11,7 +11,7 @@
 
 # Logging configuration
 [loggers]
-keys = root,sqlalchemy,alembic
+keys = root,sqlalchemy,alembic,flask_migrate
```

Comment:
Need this, so we get info back from `python manage.py db migrate` 

For example if the database isn't up to date, it wont print out an error string.

Reply:
thanks for adding this.